loki_description/launch/description.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+  <arg name="gui" default="False" />
+  <param name="use_gui" value="$(arg gui)"/>
+  <param name="robot_description" textfile="$(find magni_description)/urdf/loki.urdf" />
```

Comment:
Seems to reference magni_description directory

Reply:
Fixed, also greped for magni and removed all references to it.